src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/steps/Progress2.tsx

```diff
@@ -11,40 +11,26 @@ export function Progress2({ order, refund, cancellation }: EthFlowStepperProps)
     const isFilled = state === SmartOrderStatus.FILLED
     const isCreating = state === SmartOrderStatus.CREATING
     const isTerminalState = isRefunded || isCancelled || isFilled
+    const isInvalid = state === SmartOrderStatus.INVALID
+
+    if (isInvalid) {
+      return { status: 'error', value: 0 }
```

Comment:
Should these status values be defined in a enum?

Reply:
It's defined [here](https://github.com/cowprotocol/cowswap/blob/bf11c21cddc19dd86263e283d6fecd409c6f323e/src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/StatusIcon.tsx#L6) and I think this is fine.